### PR TITLE
[FIX] Add CBV and phase to Entity table

### LINKED ISSUE #311: Functional modalities CBV and phase are not represented in Entity table
It doesn't look like the Entity table was updated with the CBV and phase functional modalities when they were added to the specification. I think they just need to be listed under func with bold, sbref, and events.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #311. Adds the functional MRI modalities CBV and phase to the Entity table. Also fixes a couple of minor typos in the MRI specification page.

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -214,9 +214,9 @@ based fMRI a corresponding task events file (see below) MUST be provided
 multiband acquisitions, one MAY also save the single-band reference image as
 type `sbref` (e.g. `sub-control01_task-nback_sbref.nii.gz`).
 
-Each task has a unique label MUST only include of letters and/or numbers (other
-characters including spaces and underscores are not allowed). Those labels MUST
-be consistent across subjects and sessions.
+Each task has a unique label that MUST only consist of letters and/or numbers
+(other characters including spaces and underscores are not allowed). Those
```
Comment (effigies) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: Commas, and start the next sentence on a newline.

```suggestion
(other characters, including spaces and underscores, are not allowed).
Those
```

**Comment by effigies** [comment]
Thanks @tsalo. Apparently the pipes are now out of alignment. Can you remove ten spaces to make up for the ten characters you added?

**Comment by tsalo** [comment]
@effigies Thanks! It should be fixed now.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -214,9 +214,10 @@ based fMRI a corresponding task events file (see below) MUST be provided
 multiband acquisitions, one MAY also save the single-band reference image as
 type `sbref` (e.g. `sub-control01_task-nback_sbref.nii.gz`).
 
-Each task has a unique label MUST only include of letters and/or numbers (other
-characters including spaces and underscores are not allowed). Those labels MUST
-be consistent across subjects and sessions.
+Each task has a unique label that MUST only consist of letters and/or numbers
+(other characters, including spaces and underscores, are not allowed).
+Those
```
Comment (oesteban) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]:
```suggestion
Those labels MUST be consistent across subjects and sessions.
```

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -214,9 +214,10 @@ based fMRI a corresponding task events file (see below) MUST be provided
...
+labels MUST be consistent across subjects and sessions.
```
Comment (oesteban) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]:
```suggestion
```

**Comment by tsalo** [comment]
@effigies do you know if `bids-validator` is regularly updated based on changes to the specification, or should I open an issue about supporting phase data on that repository?

**Comment by effigies** [comment]
In theory it should, but if it's not detecting something, please open issues.

**Pull request merged**